### ISSUE TITLE
Question: Does built-in function make this library now redundant?

### ISSUE THREAD

**Issue by frankkienl** [Tlaster/SafeArea#2]
I've found this pull request, that adds WindowInsets support to Compose Multiplatform;
This may make this library redundant. Oh am I missing something?
https://github.com/JetBrains/compose-multiplatform-core/pull/577

**Comment by Tlaster** [comment]
Yes, this library is before that pr, after that you can just use built in WindowInsets.
